packages/jest-changed-files/src/sl.ts

```diff
@@ -34,19 +33,7 @@ const adapter: SCMAdapter = {
     }
     args.push(...includePaths);
 
-    let result: execa.ExecaReturnValue;
-
-    try {
-      result = await execa('sl', args, {cwd, env});
-    } catch (error) {
-      if (types.isNativeError(error)) {
-        const err = error as execa.ExecaError;
-        // TODO: Should we keep the original `message`?
-        err.message = err.stderr;
-      }
-
-      throw error;
-    }
+    const result = await execa('sl', args, {cwd, env});
```

Comment:
can we make the same change for all the VCS checks? trying to silence the error doesn't really seem useful in this case

Reply:
Made the change, added changelog entry!